services/controllerhost/queueDepth.go

```diff
@@ -93,16 +95,32 @@ type queueDepthCalculator struct {
 	*extentStateMonitor
 	ll                      bark.Logger
 	destinationExtentsCache replicaStatsMRUCache // MRU cache of extent replica stats
-	dlqExtentsCache         replicaStatsMRUCache // MRU cache of dlq extent replica stats
+	dstCoverMap             extentCoverMap
```

Comment:
can you add some comments on what these coverMaps are ? Why are they being reset inside processDestination() ?